sass/components/_search-form.scss

```diff
@@ -82,3 +82,65 @@ Style guide: siteheader.searchform
     min-height: 2.375em;
   }
 }
+
+/*
+Search form for internal search
+
+Markup: html/search-form-improved.html
+
+.search-form--improved
+
+Style guide: searchform.search-form--improved
+*/
+
+.search-form--improved {
+  background: #f3f3f3;
+  padding: 1.2em 1.5em;
+
+  h5 {
+    font-size: $font-size-medium;
+    text-transform: none;
+    padding-bottom: 1.2em;
+  }
+
+  .search-form__textfield-wrapper {
+    float: none;
+    width: 100%;
+    background: #fff;
+    border: 3px solid #fff;
+    border-bottom: none;
+    @extend .icon;
+
+    &:before {
+      @extend .icon--search;
+      position: absolute;
+      top: 25%;
+      left: 1%;
+      z-index: 3;
+      color: $grey;
+    }
+
+    &:hover {
+      &:before {
+        display: none;
+      }
+    }
+  }
+
+  .form-control {
+    position: relative;
```

Comment:
I think this `position: relative;` is redundant, this is already applied on _input-groups.scss